CHANGELOG.md

```diff
@@ -16,6 +16,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 * `b2sdk.v1.sync` refactored to reflect `b2sdk.sync` structure
 * Make `B2Api.get_bucket_by_id` return populated bucket objects in v2
+* `FileVersion` renamed to `LocalSyncFileVersion`
+* `B2FileVersion` renamed to `B2SyncFileVersion`
+* `File` renamed to `LocalFile` and introduced `AbstractFile` as a basis for it 
+* `file_version_info` parameter renamed everywhere to `file_version`  
+* `FileVersionInfo` renamed to `FileVersion`
+* `FileVersionInfoFactory` renamed to `FileVersionFactory` 
```

Comment:
there is so much of this stuff now that we should have a doc page which gathers information about difefrent file version classes and explains the differences between them, tells where to use what, how to specialize and roll out your own class, how to compare them etc

Reply:
We could do that. However, outside of Sync, we only have `FileVersion` and `FileVersionFactory`. The other classes are actually parts of internal API of Sync. SDK users don't really have a way to inject their own implementations of `LocalSyncFileVersion` and `B2SyncFileVersion` other than reimplementing `LocalFolder` and `B2Folder`. And since Sync will possibly be moved out to a different repo, do you think it's a good idea to document this dependancy tree?